src/formatters.rs

```diff
@@ -38,7 +39,12 @@ fn convert_wkt_to_wkb_string(s: &Series) -> PolarsResult<Option<Series>> {
 
     let wkb_string_series: Vec<String> = wkb_series
         .into_iter()
-        .map(|v| v.iter().map(|v| format!("{v}")).collect::<String>())
+        .map(|v| {
+            v.iter().fold(String::new(), |mut acc, s| {
+                let _ = write!(acc, "{s}");
```

Comment:
Might be better to unwrap the `Result` here if it is infallible and add a comment to justify.
```suggestion
                write!(acc, "{s}").unwrap();
```

Reply:
Refactored to handle possibility of non-UTF-8 bytes https://github.com/Urban-Analytics-Technology-Platform/popgetter-cli/pull/57/commits/8c8b9a1026e35d7adad488308c38bcd6f8aa50cb